assets/src/frontendIndex.js

```diff
@@ -24,3 +28,6 @@ document.querySelectorAll( `[data-render]` ).forEach(
     wp.element.render( <BlockFrontend { ...attributes.attributes } />, blockNode );
   }
 );
+
+// Other non-react initializations
+setupMediaElementJS();
```

Comment:
This will add to the render blocking time of `frontendIndex.js`. Do we have an alternative?

Reply:
These scripts are not blocking (as far as I see), they are dynamically added to the body (well, the style is attached to the head), way after the first contentful paint, the blocking scripts are pretty much the ones we are enqueueing from PHP (e.g.: the files for "slick"), as seen here:

![blockingscripts](https://user-images.githubusercontent.com/340766/96749021-ae9aae80-13a0-11eb-92a2-b49d49b31ea5.gif)

The CSS/JS files for MediaElement are not shown as blocking in the Lighthouse report.

Reply:
Maybe if we merge the Timeline PR first, then we could do these things with `useStylesheet` and `useScript` instead.

Reply:
Sounds good, I'll move those two hooks outside the `Timeline` dir in the Timeline PR.